Documentation/ExtensionArchitecture/Extbase/Reference/Validation/Validators/Index.rst

```diff
@@ -0,0 +1,624 @@
+    `elementType` is set.
+
+..  _extbase-validator-conjunction:
+
+ConjunctionValidator
+====================
+
+The `ConjunctionValidator` allows you to combine multiple validators into a
+**logical AND**. All validators in the conjunction must return valid results
+for the overall validation to pass.
+
+This validator is typically **used internally** by Extbase when multiple
+`#[Validate]` attributes are defined on a property or when validator
+conjunctions are configured in the validator resolver.
+
+Behavior:
+
+-   All validators in the conjunction are applied to the value.
+-   If **any validator fails**, the entire validation fails.
+-   Errors from all failing validators are combined in the result.
+
+While this validator is often constructed internally, you can also define your
+own validator combinations manually in the validator resolver or via custom
+validators.
+
+..  include:: _NoExamples.rst.txt
+
+..  note::
+    If you use multiple `#[Validate(...)]` attributes on a single
+    property, Extbase automatically applies them using a `ConjunctionValidator`.
+    You do not need to instantiate it manually in most use cases.
+
+..  _extbase-validator-datetime:
+
+DateTimeValidator
+=================
+
+Ensures a value is a valid \DateTimeInterface.
+
+..  code-block:: php
+
+    #[Validate(['validator' => 'DateTime'])]
+    protected mixed $startDate;
+
+..  _extbase-validator-disjunction:
+
+DisjunctionValidator
+====================
+
+The `DisjunctionValidator` is a composite Extbase validator that allows you to
+combine multiple validators using a **logical OR**.
+
+It is the inverse of the `ConjunctionValidator`: the value is considered valid
+if **at least one** of the nested validators succeeds.
+
+**Behavior:**
+-   All validators are evaluated in order.
+-   Validation stops as soon as one validator passes.
+-   If **all validators fail**, their errors are merged and returned.
+-   If **any validator passes**, the result is considered valid.
+
+**Use cases:**
+
+Use this validator when a value is allowed to match **one of multiple conditions**.
+For example:
+-   A field can be either empty or a valid email
+-   A string can be either a number or "N/A"
+-   A value can match one of multiple formats
+
+**Usage:**
+
+This validator is typically used manually in custom validators or
+in validator resolver configurations.
+
+..  include:: _NoExamples.rst.txt
+
+..  note::
+
+    Extbase does **not** automatically construct disjunctions for you.
+    You must manually create and configure a `DisjunctionValidator` when needed.
+
+    It is not currently possible to use this validator directly via
+    `#[Validate(...)]` annotations.
+
+..  _extbase-validator-emailaddress:
+
+EmailAddressValidator
+=====================
+
+The :php-short:`\TYPO3\CMS\Extbase\Validation\Validator\EmailAddressValidator`
+an email address using method :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::validEmail()`.
```

Comment:
We should link to the according `MAIL/validators` section in TYPO3_CONF_VARS here.

Reply:
done